client/state/initial-state.js

```diff
@@ -82,7 +83,13 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 			.catch( ( setError ) => {
 				debug( 'failed to set redux-store state', setError );
 			} );
-	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } ) );
+	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } );
+
+	if ( global.window ) {
```

Comment:
Isn't that a node thing? https://nodejs.org/api/globals.html#globals_global I know we're e.g. [mocking `global.window`  in some unit tests](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/test/selectors.js#L2325-L2336) to simulate a browser env.